dc/core/CMakeLists.txt

```diff
@@ -47,14 +47,10 @@ endif()
 list(APPEND DCCORE_PUBLIC_HEADERS
   ${COMMON_INCLUDES}
   types.h
-  Content.h
   ContentFactory.h
   ContentLoader.h
   ContentType.h
   ContentInteractionDelegate.h
-  ContentWindow.h
```

Comment:
Yes, they were references twice: in DCCORE_PUBLIC_HEADERS and in DCCORE_MOC_HEADERS. Now they are only in DCCORE_MOC_PUBLIC_HEADERS... :-)